src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java

```diff
@@ -32,15 +32,170 @@
 import java.util.stream.Collectors;
 
 /**
- * Funcotator (FUNCtional annOTATOR) performs functional analysis on given variants
- * and reports output in a specified output file.
+ * Funcotator (FUNCtional annOTATOR) analyzes given variants for their function (as retrieved from a set of data sources) and produces the analysis in a specified output file.
  *
- * This tool is the GATK analog of the Oncotator.
+ * <p>
```

Comment:
This is great.  Thanks.  

At some point, we may need to put a forum post for this too.

Reply:
Yup.  That's next on my list.